earth_enterprise/src/common/khCache.h

```diff
@@ -280,6 +287,22 @@ class khCache {
     }
     CheckListInvariant();
   }
+
```

Comment:
This function can be deleted. It was apparently deleted on one branch but not the other.

Reply:
Which was that other branch?  I'm still seeing it on  release_5.2.5.1_experimental